go.mod

```diff
@@ -1,6 +1,6 @@
 module go.goms.io/fleet
 
-go 1.22.3
+go 1.22.2
 
 require (
 	github.com/Azure/azure-sdk-for-go/sdk/azcore v1.9.0
```

Comment:
also bump the sdk libraries. they have many improvements to the handling of responses, pollers, auth, etc...